packages/reactivity/src/collectionHandlers.ts

```diff
@@ -331,6 +340,36 @@ const [
   shallowReadonlyInstrumentations
 ] = /* #__PURE__*/ createInstrumentations()
 
+const mapNeedToIgnoreInstrumentationKeys = ['add']
```

Comment:
```
const o = reactive(new Map())
// @ts-expect-error
o.forEach = 1
console.log(o.forEach)
console.log(o.forEach === 1) 
```
This will also happen to overwritten methods such as forEach.
I think these function names should be overwritten, and a warning should be issued while allowing the user to modify. If he actually modifies the function proxied by the responsive proxy object, it may cause responsive functional errors.